packages/docusaurus-plugin-openapi-docs/src/markdown/createResponseSchema.ts

```diff
@@ -636,7 +695,7 @@ function createEdges({
   }
 
   if (schema.oneOf !== undefined || schema.anyOf !== undefined) {
-    return createDetailsNode(
+    return createOneOfProperty(
```

Comment:
@sserrata I've just encountered a bug here while upgrading to v1.7.3. `createOneOfProperty` is called when `schema.anyOf !== undefined`, but then `createOneOfProperty` assumes that `oneOf` is defined. This causes the following error to be thrown:
```
[ERROR] TypeError: Cannot read properties of undefined (reading 'map')
    at createOneOfProperty (docusaurus-plugin-openapi-docs/lib/markdown/createResponseSchema.js:503:59)
    at createEdges (docusaurus-plugin-openapi-docs/lib/markdown/createResponseSchema.js:569:16)
    at docusaurus-plugin-openapi-docs/lib/markdown/createResponseSchema.js:110:16
    at Array.map (<anonymous>)
    at createProperties (docusaurus-plugin-openapi-docs/lib/markdown/createResponseSchema.js:109:46)
    at createNodes (docusaurus-plugin-openapi-docs/lib/markdown/createResponseSchema.js:640:24)
    at docusaurus-plugin-openapi-docs/lib/markdown/createResponseSchema.js:774:67
    at Array.map (<anonymous>)
    at createResponseSchema (docusaurus-plugin-openapi-docs/lib/markdown/createResponseSchema.js:710:33)
    at docusaurus-plugin-openapi-docs/lib/markdown/createStatusCodes.js:290:91
```

I can work on a Pull Request to resolve next week unless you want to handle it.